Mapped a few new things

Darkhax opened the pull request:
Just mapped out the basic objects used for resource indexing. 
Edit: decided to map a few other things as well

Darkhax committed:
Mapped resource index objects.

Darkhax committed:
Mapped a bunch of misc stuff.

Darkhax committed:
Mapped something to do with default light values.

asiekierka commented:
You might want to mark this WIP if you're still commiting.

darkhax commented:
Nope, not WIP. Nobody had reviewed it, and it, and I had a few minutes, so I decided to add a few things earlier. I prefer to do one PR per mapping session, as it minimizes the risk of merge conflicts.

asiekierka commented:
Ah!

2016-10-13 11:45 GMT+02:00 Tyler Hancock <notifications@github.com>:

> Nope, not WIP. Nobody had reviewed it, and it, and I had a few minutes, so
> I decided to add a few things earlier. I prefer to do one PR per mapping
> session, as it minimizes the risk of merge conflicts.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/FabricMC/pomf/pull/108#issuecomment-253466505>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAG7ar1qOUJBsexcP1WAu_9_PeAJB5Raks5qzf22gaJpZM4KVHjV>
> .
>

copygirl commented on mappings/net/minecraft/client/render/entity/EntityRendererArrows.mapping:
```diff
@@ -0,0 +1,7 @@
+CLASS none/bvg net/minecraft/client/render/entity/EntityRendererArrows
```
`Arrows` ? Shouldn't it just be `Arrow` ?

Darkhax replied:
This layer renders all of the arrows sticking out of the entity.

copygirl replied:
That's confusing then, how about `StuckArrows`? I guess it could _still_ be confused for allows stuck in the ground or so but at least it's a step in the right direction, imo.

copygirl commented on mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping:
```diff
@@ -0,0 +1,10 @@
+CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
+	FIELD a OVERLAY_TEXTURE Lnone/kp;
```
`OVERLAY_TEX` or just `TEX` if it's a simple 2D texture. Should be `SKIN` if it's applied to a 3D model.

Darkhax replied:
Is that a universal naming scheme for entity textures? In this case the texture is `textures/entity/skeleton/stray_overlay.png`

copygirl replied:
`SKIN` is currently the standard for anything that goes on a 3D model. If we decide it should be something different, we can easily search/replace all occurrences.

copygirl commented on mappings/net/minecraft/world/EnumLightTypes.mapping:
```diff
@@ -0,0 +1,6 @@
+CLASS none/ajc net/minecraft/world/EnumLightTypes
+	FIELD a SKY Lnone/ajc;
+	FIELD b SOLID Lnone/ajc;
```
`SOLID` ? Is that block light?

Darkhax replied:
Yes, this is block light. Wasn't sure if it should be called BLOCK or SOLID.

copygirl replied:
Unless we want to call "block light" something different in pomf, I think it should be `BLOCK`.

Darkhax committed:
Fixed several issues

- Stray overlay texture is now SKIN
- Mapped the missing field for stray model.
- Renamed EntityrRendererArrows -> EntityRendererStuckArrows
- Renamed EnumLightTypes -> LightType

darkhax commented:
Just fixed the earlier issues. Here is an overview of my changes. 
- Stray overlay texture is now SKIN
- Mapped the missing field for stray model.
- Renamed EntityrRendererArrows -> EntityRendererStuckArrows
- Renamed EnumLightTypes -> LightType

copygirl commented on mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping:
```diff
@@ -1,6 +1,7 @@
 CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
-	FIELD a OVERLAY_TEXTURE Lnone/kp;
+	FIELD a STRAY Lnone/kp;
```
We haven't used the name of the thing so far and I don't think we should start. My vote is for `SKIN`, though `SKIN_OVERLAY` is an alternative.

Darkhax replied:
Sorry, skin is what I intended to write. Will fix.

Darkhax committed:
Fixed a typo
